packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -507,7 +569,17 @@ void main() {
       ]
     ));
 
-    await Dart2JSTarget(WebRendererMode.auto).build(environment);
+    await Dart2JSTarget(
+      const JsCompilerConfig(
+        csp: false,
+        dumpInfo: false,
+        nativeNullAssertions: false,
+        noFrequencyBasedMinification: false,
+        optimizationLevel: 'O4',
+        sourceMaps: false,
+        renderer: WebRendererMode.auto,
+      )
```

Comment:
A similar though came to my mind about this, but then I got lazy. What I really wanted to do is to just have default values in the constructor, so that you can only specify the properties that diverge from the defaults. I think I'll actually do that in this PR, that's probably a good change.

Reply:
The default values make the test initializations much more compact, thanks!